### PR TITLE
Most items

### PR DESCRIPTION
**Pull request by wlffann**
This implements the `most_items` endpoint that returns the top items sold by quantity.

All Rspec tests passing, related tests in spec harness passing.

@case-eee @drod1000

### CODE REVIEW

File: app/controllers/api/v1/most_items_controller.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class Api::V1::MostItemsController < ApplicationController 
+
+  def index
+    render json: Item.joins(:invoice_items)
```
Comment (case-eee) [inline app/controllers/api/v1/most_items_controller.rb:4]: @wlffann woooo business intelligence method 🎉  can we move this logic out of the controller though? where might be more appropriate?

**Review by case-eee — COMMENTED** [review]
@wlffann @drod1000 see my one comment here. good work!

File: app/controllers/api/v1/most_items_controller.rb (hunk 1 above)
Comment (wlffann) [reply app/controllers/api/v1/most_items_controller.rb:4]: @case-eee Good question! I'll see if I can move it down to the model level.

Comment (case-eee) [reply app/controllers/api/v1/most_items_controller.rb:4]: awesome @wlffann 👍

**Pull request merged**